pelutils/jsonl.py

```diff
@@ -25,7 +25,7 @@ def dump(objects: Iterable, f: TextIO, single_block=True):
     If single_block is True, objects will be joined to a single block before being written.
     It can be usefull to set this to False if there is a large amount of lazily generated data. """
     if single_block:
-        f.write(os.linesep.join(rapidjson.dumps(obj) for obj in objects))
+        f.write(os.linesep.join(rapidjson.dumps(obj) for obj in objects) + "\n")
```

Comment:
Vær lige konsekvent og ordentlig for en gangs skyld i dit liv og brug os.linesep